gradle.properties

```diff
@@ -20,29 +20,29 @@ kotlin.code.style=official
 org.gradle.jvmargs=-Xms1024m -Xmx4096m
 # Fabric Properties
 # Check these on https://fabricmc.net/versions.html
-minecraft_version=1.21.1
-yarn_mappings=1.21.1+build.3
+minecraft_version=1.21.4
+yarn_mappings=1.21.4+build.1
 loader_version=0.16.9
 min_loader_version=0.16.5
 
 # Fabric API
-fabric_version=0.110.0+1.21.1
+fabric_version=0.111.0+1.21.4
 # Loom
-loom_version=1.9-SNAPSHOT
+loom_version=1.8-SNAPSHOT
 # Mod Properties
 mod_version=0.22.1
 maven_group=net.ccbluex
 archives_base_name=liquidbounce
 # Kotlin
 # https://kotlinlang.org/releases.html
-kotlin_version=2.1.0
+kotlin_version=2.0.21
 # https://maven.fabricmc.net/net/fabricmc/fabric-language-kotlin/
-fabric_kotlin_version=1.13.0+kotlin.2.1.0
+fabric_kotlin_version=1.12.3+kotlin.2.0.21
```

Comment:
Why are we downgrading Kotlin?

Reply:
I'll take care of the versions as the last step.